.azure-pipelines/pr_test_scripts.yaml

```diff
@@ -239,11 +239,12 @@ multi-asic-t1-lag:
   - process_monitoring/test_critical_process_monitoring.py
   - container_checker/test_container_checker.py
   - http/test_http_copy.py
+  - generic_config_updater/test_multiasic_scenarios.py
```

Comment:
Just to confirm, is it a T1 case or T2 case?

Reply:
In general, it is for multi asic, but since T2 has not been ready yet, hence added int multi asic T1 category.